corehq/apps/app_manager/app_strings.py

```diff
@@ -61,6 +64,16 @@ def maybe_add_index(text):
                 yield id_strings.detail_tab_title_locale(module, detail_type, tab), trans(tab.header)
 
         yield id_strings.module_locale(module), maybe_add_index(trans(module.name))
+        if hasattr(module, 'report_configs'):
+            for config in module.report_configs:
+                yield id_strings.report_command(config.report_id), trans(config.header)
+                yield id_strings.report_name(config.report_id), config.report.title
+                yield id_strings.report_menu(), 'Reports'
+                yield id_strings.report_name_header(), 'Report Name'
+                yield id_strings.report_description_header(), 'Report Description'
```

Comment:
Yeah I think since we don't have this in the UI yet, we're just hard-coding for how. This will support translation similar to the "Case List Label", etc for normal modules